pkg/portal/regions.go

```diff
@@ -0,0 +1,93 @@
+package portal
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+)
+
+var (
+	PROD_REGIONS = []string{
```

Comment:
I looked through the repo and couldn't find anywhere else we're defining or using a list like this, but if the need arises anywhere else it would probably be a good idea to pull this out to a central location so we only have to change one place when we add a new region. No change needed for now, just wanted to make a note of it.

Reply:
A ticket has been created to create an api for our cloud regions which this will eventually be pointed at

Reply:
Until then can we add a note in our region onboarding doc to make sure we update this list? [region rollout](https://msazure.visualstudio.com/AzureRedHatOpenShift/_wiki/wikis/ARO.wiki/59387/New-region-rollout)

Reply:
Absolutely, I will do this prior to merging